### DISCUSSION TITLE
Noob here! How do you download Trillium and open it offline with no host?

### DISCUSSION THREAD

**Discussion by LuciusDuck** [TriliumNext/Trilium#4451] [Q&A] [answered]
What is a docker? Do I need one? Is it an exterior host? I want offline completely. Thanks in advance! Beautiful Software you've got here. Beautiful. Near everything I could've asked for.

**Comment by zerebos** [comment]
If you just want an offline-only note taking app, just download the correct release for your OS (windows/mac/linux) from the releases page. Open it up and you're all set! No need to setup a sync server or anything to do with docker.

**Reply by LuciusDuck** [reply]
Great! Thank you Rauenzi. I downloaded and unzipped it, however, within the Trillium-Master folder, I cannot find a file that upon clicking it, launches the app.

I downloaded it from this page: https://github.com/zadam/trilium/tree/v0.61.14
And downloaded the ZIP.

**Reply by zerebos** [reply, marked as answer]
That downloads the codebase, you should be downloading for your OS on the releases page, see here: https://github.com/zadam/trilium/releases/latest

**Reply by LuciusDuck** [reply]
Thank you very much!
